it/pages/mobileNumber/CaptureMobileNumberPageSpec.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.mobileNumber
+
+import common.SessionKeys.validationMobileKey
+import controllers.mobileNumber.routes
+import forms.MobileNumberForm.mobileNumberForm
+import helpers.SessionCookieCrumbler
+import pages.BasePageISpec
+import play.api.http.Status
+import play.api.libs.ws.WSResponse
+import stubs.VatSubscriptionStub._
+
+class CaptureMobileNumberPageSpec extends BasePageISpec {
+
+  val path = "/new-mobile-number"
+  val newMobile = "01952654321"
+
+  "Calling the Capture mobile number (.show) route" when {
+
+    def show: WSResponse = get(path, formatInflightChange(Some("false")))
+
+    "the user is authenticated" when {
+
+      "a success response with contact numbers is received from Vat Subscription" should {
+
+        "load successfully" in {
+
+          given.user.isAuthenticated
+          stubCustomerInfo
+
+          val result = show
+
+          result should have(
+            httpStatus(Status.OK),
+            pageTitle(generateDocumentTitle("captureMobile.title"))
+           )
+        }
+
+        "add the existing mobile number to session" in {
+
+          given.user.isAuthenticated
+          stubCustomerInfo
+
+          val result = show
+
+          SessionCookieCrumbler.getSessionMap(result).get(validationMobileKey) shouldBe Some(currentMobile)
+        }
+      }
+    }
+  }
+
+  "Calling the Capture mobile number (.submit) route" when {
+
+    def submit(data: String): WSResponse = post(path, Map(
+      validationMobileKey -> currentMobile) ++ formatInflightChange(Some("false"))
+    )(toFormData[String](mobileNumberForm(currentMobile), newMobile))
+
+    "the user is authenticated" when {
+
+      "a valid mobile number has been submitted" should {
+
+        "redirect to the the Confirm Contact Numbers page" in {
```

Comment:
Can you put "redirect to the Confirm Mobile Number page" and also please update the same test name in the CaptureLandlineNumberPageSpec please? It was missed on the previous review by the looks of it.